templates/dashd.conf.template

```diff
@@ -1,55 +1,67 @@
 # general
-
-devnet=evonet-6
-
 daemon=0  # leave this set to 0 for Docker
 logtimestamps=1
 maxconnections=256
-
-# optional indices (required for Insight)
-txindex=1
-addressindex=1
-timestampindex=1
-spentindex=1
-
-# ZeroMQ notifications (required for Insight)
-zmqpubrawtx=tcp://0.0.0.0:29998
-zmqpubrawtxlock=tcp://0.0.0.0:29998
-zmqpubhashblock=tcp://0.0.0.0:29998
+printtoconsole=1
 
 # JSONRPC
 server=1
-rpcport=20002
-rpcuser=dashrpc
-rpcpassword=password
+rpcuser={{=it.core.rpc.user}}
+rpcpassword={{=it.core.rpc.password}}
 
-rpcallowip=127.0.0.0
+rpcallowip=127.0.0.1
 rpcallowip=172.16.0.0/12
-rpcallowip=172.27.0.0/16
 rpcallowip=192.168.0.0/16
 
 rpcworkqueue=64
 rpcthreads=16
 
 # external network
 listen=1
-bind=0.0.0.0
 dnsseed=0
 allowprivatenet=0
 
-printtoconsole=1
+{{? it.network.name === 'testnet'}}
+{{??}}
+# optional indices (required for Insight)
+txindex=1
+addressindex=1
+timestampindex=1
+spentindex=1
 
-addnode=seed-1.evonet.networks.dash.org:20001
-addnode=seed-2.evonet.networks.dash.org:20001
-addnode=seed-3.evonet.networks.dash.org:20001
-addnode=seed-4.evonet.networks.dash.org:20001
-addnode=seed-5.evonet.networks.dash.org:20001
+# ZeroMQ notifications (required for Insight)
+zmqpubrawtx=tcp://0.0.0.0:29998
+zmqpubrawtxlock=tcp://0.0.0.0:29998
+zmqpubhashblock=tcp://0.0.0.0:29998
+{{?}}
 
-sporkaddr=yQuAu9YAMt4yEiXBeDp3q5bKpo7jsC2eEj
+
+{{? it.network.name === 'local'}}
+regtest=1
+[regtest]
+{{?? it.network.name === 'evonet'}}
+devnet={{=it.network.name}}{{? it.network.version !== null}}-{{=it.network.version}}{{?}}
+{{? it.core.docker.image.includes('16') }}[devnet]{{?}}
```

Comment:
endsWith?

Reply:
OK, testing that it is not `15` for now. Unit testing is usually used to make sure code works properly, right? How does that work, do we use it anywhere?

Reply:
Yeah, but in this case it's more like functional test when we are testing against a blackbox. We have unit and integration tests in almost all projects/repos. You can check Drive, DPP, or DAPI for example.

Reply:
I feel we could remove this legacy. We will update testnet/evonet to 0.16 soon so we don't need to support 15 version anymore.